app/views/products/index.html.haml

```diff
@@ -0,0 +1,12 @@
+%nav.navbar.navbar-default{:role => "navigation"}
+      %ul.nav.navbar-nav
```

Comment:
make sure that you have the same tab size in all files

Reply:
Corrected!